api-doc.md

```diff
@@ -328,3 +328,33 @@
     - Headers : none
     - Message : Leave request has been sent successfully
 ```
+
+### 5. Send Update Leave Request mail
+
+```
+- Method : POST
```

Comment:
Let's use `PUT` method for update

Reply:
This send the response mail, not updates the data. Should we use PUT method?